matchms/Pipeline.py

```diff
@@ -132,25 +149,22 @@ def run(self):
         # Processing
         self.write_to_logfile("--- Processing spectra ---")
         self.write_to_logfile(f"Time: {str(datetime.now())}")
-        for step in self.filter_steps_queries:
-            self.write_to_logfile(f"-- Processing step: {step} --")
-        for i in tqdm(range(len(self.spectrums_queries)),
-                             disable=(not self.progress_bar),
-                             desc="Processing query spectrums"):
-            spectrum = self.spectrums_queries[i]
-            for step in self.filter_steps_queries:
-                spectrum = self.apply_filter(spectrum, step)
-            self.spectrums_queries[i] = spectrum
-        self.spectrums_queries = [s for s in self.spectrums_queries if s is not None]
+        # Process query spectra
+        spectrums, report = self.processing_queries.process_spectrums(
+            self.spectrums_queries,
+            create_report=True,
+            progress_bar=self.progress_bar)
+        self.spectrums_queries = spectrums
+        self.write_to_logfile(str(report))
```

Comment:
In principle that makes sense. But I do not have a quick fix for this because the Pipeline logging file has a somewhat different logic than the normal matchms logger (for me at least). With the normal logger I have to decide the logging level, but while I would under all circumstances add the created report to the logging file it somehow doesn't fit to the level descriptions such as "warning" or higher. With the method called here, I am sure it ends up in the logging file no matter if user set the threshold to info, warning, etc.

I guess it best is made into a new issue?